br/backup-and-restore-tool.md

```diff
@@ -246,10 +265,95 @@ The `table` sub-command has two options:
+
+You can refer to an [AWS Official Document](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-bucket.html) to create an S3 `Bucket` in the specified `Region`. You can also refer to another [AWS Official Document](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-folder.html) to create a `Folder` in the `Bucket`.
+
+Pass `SecretKey` and `AccessKey` of the account that has permission to access the S3 backend to the BR node. Here `SecretKey` and `AccessKey` are passed as environment variables. And then pass the permission to the TiKV node through BR.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+export AWS_ACCESS_KEY_ID=${AccessKey}
+export AWS_SECRET_ACCESS_KEY=${SecretKey}
+```
+
+When backing up using BR, explicitly specify the parameters `--s3.region` and `--send-credentials-to-tikv`. `--s3.region` indicates the region where S3 is located, and `--send-credentials-to-tikv` means passing the privilege to access S3 to the TiKV node.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+br backup full \
+    --pd "${PDIP}:2379" \
+    --storage "s3://${Bucket}/${Folder}" \
+    --s3.region "${region}" \
+    --send-credentials-to-tikv=true \
+    --log-file backuptable.log
+```
+
+### Back up incremental data
+
+If you want to back up incrementally, you only need to specify the **last backup timestamp** `--lastbackupts`.
+
+The following are some limitations of the incremental backup:
+
+- The incremental backup needs to be under a different path from the previous full backup.
+- No GC (Garbage Collection) happens between the start time of the incremental backup and `lastbackupts`.
+
+Execute the following command to back up the incremental data between `(LAST_BACKUP_TS, current PD timestamp)`:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+br backup full\
+    --pd ${PDIP}:2379 \
+    -s local:///home/tidb/backupdata/incr \
+    --lastbackupts ${LAST_BACKUP_TS}
+```
+
+Execute the `validate` command to get the timestamp of the last backup. For example:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/backupdata`
+```
+
+In the above example, the incremental backup data includes the written data and the DDLs between `(LAST_BACKUP_TS, current PD timestamp]`. When restoring data, BR restores DDLs first, and then restore the written data.
+
+### Back up Raw KV (experimental feature)
+
+> **Warning:**
+> 
+> This feature is in the experiment, without being thoroughly tested. It is highly **not recommended** to use this feature in the production environment.
+
+In some scenarios, TiKV might run independently of TiDB. Given that, BR also supports bypassing the TiDB layer and backing up data in TiKV.
+
+For example, you can execute the following command to back up all keys between `[0x31, 0x3130303030303030)` of the default CF to `$BACKUP_DIR`:
```

Comment:
IMO, since CF = column family, `in` might be better.
```suggestion
For example, you can execute the following command to back up all keys between `[0x31, 0x3130303030303030)` in the default CF to `$BACKUP_DIR`:
```